docs/further-reading/create-config.md

```diff
@@ -0,0 +1,70 @@
+---
+id: create-config
+title: Create Config
+---
+The `centrifuge createconfig` command automatically generates all required key pairs and config files
+to run the node.
+
+  ```bash
+  $ centrifuge createconfig -z  ~/.ethereum/keystore/<KEY-FILE> -e ws://127.0.0.1:8546 -t <DEFINE_CONFIG_DIR_NAME> \
+  -a 8082 -p 38204
+  ```
+
+Replace the `<KEY-FILE>` with the key file you obtained when creating the Ethereum account and `<DEFINE_CONFIG_DIR_NAME>` with the location where you want the `config.yaml`  file to be stored.
+
+> **NOTE:** If the provided `keystore/<KEY-FILE>` is encrypted the password needs to be added with `-z SAMPLE_PASSWORD` to successfully create the config.
+
+Per default the centrifuge network `russianhill` is using the Ethereum testnet rinkeby.
+
+
+**Flags for createconfig command:**
+
+```text
+  -z, --accountkeypath string   Path of Ethereum Account Key JSON file (default "/Users/username/datadir/main.key")
```

Comment:
`Path of Ethereum Account UTC JSON Keystore file (default "$HOME/datadir/main.key")`